src/js/controller/controller.js

```diff
@@ -649,7 +649,6 @@ define([
             this.setCurrentAudioTrack = _setCurrentAudioTrack;
             this.getCurrentAudioTrack = _getCurrentAudioTrack;
             this.getAudioTracks = _getAudioTracks;
-            this._getCurrentQuality = _getCurrentQuality;
```

Comment:
Thanks for the catch! Fixed https://github.com/jwplayer/jwplayer/commit/34d0267b93513fe58b967bc173145b258df19678